### ISSUE TITLE
TAP_TO_VPP interface is not stable after updating

### ISSUE THREAD

**Issue by artem-belov** [ligato/vpp-agent#1350]
## Current Behavior

ping doesn't pass through the connection

## Failure Information

after updating cross connect ping doesn't pass. Arp entry is incomplete.
```
/ # arp
? (10.30.2.2) at <incomplete>  on if2_src
? (10.32.0.1) at 8a:94:19:3b:5f:ed [ether]  on eth0 
```
Reverting commit https://github.com/ligato/vpp-agent/commit/8c071c08215522b46b89a26c10a18f9fa4e0d8dd resolves problem

Related to [networkservicemesh#1128](https://github.com/networkservicemesh/networkservicemesh/issues/1128)

Log is attached

## Context

* ligato/vpp-agent:v2.1.0

## Failure Logs

[nsmd-dataplane.log](https://github.com/ligato/vpp-agent/files/3207125/nsmd-dataplane.log)

**Comment by ondrej-fabry** [comment]
My quick guess is that the used if condition for checking IPv6 address is actually wrong:

https://github.com/ligato/vpp-agent/blob/400e6c3ae7ef39f78a15f8f1830f017fecce765b/plugins/linux/ifplugin/descriptor/interface.go#L366

This will return true even for IPv4 addresses, see here: https://play.golang.org/p/ZhEwkNwlGf6

Can you please test that changing the condition to `ipAddress.IP.To4() == nil` fixes this?

**Comment by artem-belov** [comment]
@ondrej-fabry yes, it fixes the issue

**Comment by artem-belov** [comment]
@ondrej-fabry Could you please update docker hub image v2.1.0 with that fix

**Comment by ondrej-fabry** [comment]
Just released [v2.1.1](https://github.com/ligato/vpp-agent/releases/tag/v2.1.1) which includes this fix.

**Comment by artem-belov** [comment]
Thank you
